packages/client-sdk-nodejs/test/unit/cache-client.test.ts

```diff
@@ -7,13 +7,9 @@ import {
   StringMomentoTokenProvider,
 } from '../../src';
 import {SimpleCacheClientProps} from '../../src/cache-client-props';
-
-// This auth token is syntactically correct but not actually valid; it won't work with the real Momento Servers.
-// Used only for unit testing the constructors etc.
-const fakeAuthTokenForTesting =
-  'eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJzcXVpcnJlbCIsImNwIjoiY29udHJvbCBwbGFuZSBlbmRwb2ludCIsImMiOiJkYXRhIHBsYW5lIGVuZHBvaW50In0.zsTsEXFawetTCZI';
 const credentialProvider = new StringMomentoTokenProvider({
-  authToken: fakeAuthTokenForTesting,
+  authToken:
+    'eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJzcXVpcnJlbCIsImNwIjoiY29udHJvbCBwbGFuZSBlbmRwb2ludCIsImMiOiJkYXRhIHBsYW5lIGVuZHBvaW50In0.zsTsEXFawetTCZI',
```

Comment:
This change snuck in because of a merge from main followed by revert. I have merged the changes from main again.
Fixed in https://github.com/momentohq/client-sdk-javascript/pull/598/commits/bd1add565f4a5c62a79606fdc11edc43904bb02e